internal/manifest/oci_index.go

```diff
@@ -57,6 +57,7 @@ func (index *OCI1IndexPublic) Instance(instanceDigest digest.Digest) (ListUpdate
 				Digest:    manifest.Digest,
 				Size:      manifest.Size,
 				MediaType: manifest.MediaType,
+				Platform:  manifest.Platform,
```

Comment:
It would be a bit safer to make a clone instead of providing a pointer to our data, but, meh, we can deal with that if it breaks I guess.